EntertainmentCost.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class EntertainmentCost {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter age: ");
+        int age = scanner.nextInt();
+        scanner.nextLine();
+
+        System.out.print("Is the customer VIP? (yes/no): ");
+        String vipStatus = scanner.nextLine().trim().toLowerCase();
+        boolean isVip = vipStatus.equals("yes");
```

Comment:
Don't use methods we haven't used before as I stated. Use only what we have learned so far and the features Scanner can provide.